src/App.jsx

```diff
@@ -1,13 +1,204 @@
-
 import './App.css'
+import Fruitbox from "./components/Fruitbox.jsx";
+import React from "react";
+import kiwi from "./assets/kiwi.jpeg"
+import appels from "./assets/appels.jpg"
+import bananen from "./assets/bananen.jpg"
+import aardbei from "./assets/aardbei.png"
+import Button from "./components/Button.jsx";
+
+import FormField from "./components/FormField.jsx";
+import Dropdown from "./components/Dropdown.jsx";
+
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [aardbeiCounter, aardbeiSetCounter] = React.useState(0);
+    const [banenenCounter, banenenSetCounter] = React.useState(0);
+    const [appelCounter, appelSetCounter] = React.useState(0);
+    const [kiwiCounter, kiwiSetCounter] = React.useState(0);
+    const [textField, setTextField] = React.useState("");
+    const [firstName, setFirstName] = React.useState("");
+    const [lastName, setLastName] = React.useState("");
+    const [age, setAge] = React.useState(0);
+    const [adress, setAdress] = React.useState("");
+    const [postcode, setPostcode] = React.useState("");
+    const [comment, setComment] = React.useState("");
+    const [radioButton, setRadioButton] = React.useState("overdag");
+    const [agreeTerms, setAgreeTerms] = React.useState('');
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log("Je bestelling wordt gemaakt");
+    }
+
+    //Is mijn handleSubmit zo juist? In de console krijg ik wel deze melding. Maar moet ik deze ook krijgen in een popup?
+    // Zijn mijn radiobuttons zo juist? deze worden gereset wanneer ik bij opmerking iets ga typen
+    // en zodra ik iets getypt heb bij opmerking, dan wordt de checkbox daaronder gereset
+    // in de console krijgi k ook een foutmelding dat de <body> niet gebruikt kan worden als children,
+    // maar ik snap niet wat ik fout doe
+    // Hoor graag in de feedback wat ik eventueel fout doe en hoe het dan wel moet, thx :)
+
+    return (
+        <body className="outer-box">
```

Comment:
body is niet een element wat je zelf hoeft toe te voegen, ik zou hier dus een div gebruiken